src/utils/bindActionCreators.js

```diff
@@ -21,6 +22,13 @@ function bindActionCreator(actionCreator, dispatch) {
  * action creator wrapped into the `dispatch` call.
  */
 export default function bindActionCreators(actionCreators, dispatch) {
+  invariant(typeof actionCreators === 'function' ||
```

Comment:
Let's move this whole check after `if (typeof actionCreators === 'function') {` condition below. Then you only need to check `!= null` because if it's a function, we won't get there anyway.